homework-submission/homeworkWeek1/app.js

```diff
@@ -0,0 +1,76 @@
+const TodoList = (props) => {
+    console.log(props)
```

Comment:
Nice using of stateless components.
And it would be nice to leave out the log of this line.